db/seeds.rb

```diff
@@ -1,249 +1,242 @@
-master_service.update_columns(deployment_option: 'self_managed')
-
-# Enable account_plans / service_plans and end_users plans for Master
-%i(end_users account_plans service_plans).each do |setting|
-  master.settings.send(setting).show!
-end
-master.settings.allow_branding!
+  master = Account.create!(name: master_name) do |account|
+    account.subdomain = master_domain
+    account.master = true
+    account.state = 'approved'
+    account.site_access_code = master_access_code
+  end
 
-# Adding default metrics for Master API
-{billing: 'Billing API', account: 'Account Management API', analytics: 'Analytics API'}.each do |system_name, description|
-  master_service.metrics.create!(system_name: system_name, unit: 'hit', friendly_name: description)
-end
+  master.update!(provider_account: master)
+
+  FieldsDefinition.create_defaults!(master)
+
+  if Rails.env.development?
+    FieldsDefinition.create!([
+                               {
+                                 account: master,
+                                   target: 'Account',
+                                   hidden: false,
+                                   required: false,
+                                   label: 'Are you interested in 3scale on-premises?',
+                                   name: 'API_Onprem_3s__c',
+                                   choices_for_views: "Yes, I'm interested in on-premises \n No, I'm only interested in SaaS"
+                               },
+                               {
+                                 account: master,
+                                   target: 'Account',
+                                   hidden: false,
+                                   required: false,
+                                   label: 'At what stage is your API Project?',
+                                   name: 'API_Status_3s__c',
+                                   choices_for_views: 'In design, In development, Live'
+                               },
+                               {
+                                 account: master,
+                                   target: 'Account',
+                                   hidden: false,
+                                   required: false,
+                                   label: 'What is the purpose of your API?',
+                                   name: 'API_Purpose_3s__c',
+                                   choices_for_views: 'Integration point for customers, Integration point for partners,' \
+      'Content/media distribution, Mobile backend, E-commerce and/or affiliate program,' \
+      'API-based business model, Company internal integration'
+                               }
+                             ])
+  end
 
-###
-#  Creating Provider Account
-###
+  master_user = master.users.create!(username: master_login, password: master_password, password_confirmation: master_password) do |user|
+    user.signup_type = 'minimal'
+    user.role = :admin
+    user.state = 'active'
```

Comment:
This will not fill `activated_at`. Better keep using `user.activate!`

Reply:
@guicassolato It fills it for sure because it is in the tests 😄

Reply:
ah right, the `activated_at` !!! will do it 👍 💪